src/libstd/sys/redox/start.rs

```diff
@@ -0,0 +1,118 @@
+use sys::syscall::exit;
```

Comment:
#### Hygiene error! In order to generate executables, we have to have a naked `_start` function at the libstd root.

This code would typically be done in `crt0` in `libc`. Since we are avoiding the linkage to a `libc`, we define it here. Due to the way `#[no_mangle]` works, the linker will fail if these functions are not exported all the way to the crate root.